### ISSUE TITLE
Feature request: Skip users verified via affiliate accounts

### ISSUE THREAD

**Issue by drummerXD** [kheina-com/Blue-Blocker#117]

**Comment by DanielleMiu** [comment]
is this something that isn't already covered by the "skip users verified via businesses (gold check)" option?

**Comment by drummerXD** [comment]
No, unless the other end of the affiliate is ALSO a gold check. 

![image](https://github.com/kheina-com/Blue-Blocker/assets/86092951/f484e318-52dd-4974-a52c-c6dacc6e668f)

In this screenshot, NHLPlayerSafety (gold + affiliate) remains, but NHLFantasy and PR_NHL (blue + affiliate) are blocked.
